### ISSUE TITLE
CGSpace fails OAI validation at base-search

### ISSUE THREAD

**Issue by alanorth** [ilri/DSpace#63]
We fixed OAI support in #57, but now, testing CGSpace's OAI with the [base-search](http://oval.base-search.net/) validation tool fails with the following single fatal error:

```
ERROR: No incremental harvesting (day granularity) of ListRecords: Harvest for reference date 2015-02-14 returned no records. Please make sure that the from and until parameters are evaluated.
```

I've seen several messages on the dspace-tech mailing list about this, and will have to investigate their solutions. //cc @Jayakananthan

**Comment by alanorth** [comment]
Testing with base-search again, we still fail this same check:

![oval-validate](https://cloud.githubusercontent.com/assets/191754/8001040/743ab3a0-0b6a-11e5-8185-b521a17c3f7e.png)

According to the discussion in [DS-2542](https://jira.duraspace.org/browse/DS-2542) it looks like the problem is in DSpace 4.x - 5.2, with a fix perhaps to be merged in 5.3. DSpace's XOAI module supports `from` and `until` dates, but returns no results if you query for one single day, like `from=2012-07-17&until=2012-07-17`.

Test: https://cgspace.cgiar.org/oai/request?verb=ListRecords&metadataPrefix=oai_dc&from=2012-07-17&until=2012-07-17

**Comment by alanorth** [comment]
DSpace Test was migrated to DSpace 5.5 and now passes validation. Pinging @Jayakananthan if he's still around! Testing from [base search](http://oval.base-search.net/) with this string:

https://dspacetest.cgiar.org/oai/request?verb=Identify
